README.md

```diff
@@ -2,13 +2,31 @@
 Listen to music from your Spotify Premium music account
 
 ## About
-Stream your favorite music from the popular Spotify music service.  Spotify
+Stream your favorite music from the popular Spotify music service. Spotify
 Premium users can search and play tracks from their own playlists or the huge
 Spotify music library.
 
 You can also control your Mycroft device using the Spotify Connect system.
 So play DJ on your phone while listening on Mycroft!
 
+### This skill doesn't do any playback
+
+This skills works with the spotify connect protocol and generally requires the user to setup a spotify player themselves.
```

Comment:
```suggestion
This skill works with the Spotify Connect protocol to interact with Spotify devices, but doesn't perform any playback itself. If you want playback on the hosting Mycroft device, you'll need to set up a player yourself.
```

Reply:
Ya this is an important aspect to consider.  Users don't HAVE to install raspotify locally, if they don't want to playback on the mycroft device itself, and rather just control other Spotify compatible players on the network.